src/analysis/normed/order/basic.lean

```diff
@@ -54,6 +55,12 @@ extends linear_ordered_field α, has_norm α, metric_space α :=
 (dist_eq : ∀ x y, dist x y = ‖x - y‖ . obviously)
 (norm_mul' : ∀ x y : α, ‖x * y‖ = ‖x‖ * ‖y‖)
 
+/-- A `normed_lattice_field` is a `normed_linear_ordered_field` that satisfies also the `solid`
+property: `|a| ≤ |b|` implies `‖a‖ ≤ ‖b‖`. -/
+class normed_lattice_field (α : Type*)
+extends normed_linear_ordered_field α :=
```

Comment:
This looks like a pretty bad name to me given that `linear_order α → lattice α` suggests `normed_linear_ordered_field α → normed_lattice_field α` instead of the other way around.

Reply:
The problem is that `normed_lattice_add_comm_group` refers nowhere to the stronger `solid` axiom. Maybe it could be renamed to `solid_lattice_add_comm_group`, at risk of being a bit obscure?

Reply:
Sure, but the name `solid` is not very popular so I am not sure about making it more present.

Reply:
I have a problem with `solid` on its own, but as part of a compound name it looks alright.